src/appleseed/foundation/utility/searchpaths.h

```diff
@@ -125,9 +133,10 @@ class SearchPaths
 
     // Return a string with all the search paths separated by the specified separator,
     // optionally making them absolute and/or listing them in reverse order.
-    std::string to_string(
-        const char separator = ':',
-        const bool reversed = false) const;
+    std::string to_string(const char separator, const bool reversed) const;
```

Comment:
Maybe having two methods would be clearer than having a boolean argument? E.g.

    to_string()
    to_string_reversed()